### DISCUSSION TITLE
Not able to upload the profile picture

### DISCUSSION THREAD

**Discussion by arnab-bit** [community/community#82948] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

I have resized the image below 1 MB. Still, I'm not able to upload the profile picture. Please fix the issue.

**Comment by MEClouds** [comment]
If you're encountering issues uploading your profile picture on GitHub even after resizing it to below 1 MB, here are some steps you can take to troubleshoot and potentially resolve the problem:

Double-Check File Format:
Ensure that your image is in a supported format (e.g., JPEG, PNG). GitHub has specific requirements for profile pictures.

Clear Browser Cache and Cookies:
Sometimes, cached data can cause issues. Try clearing your browser's cache and cookies, then attempt the upload again.

Use a Different Browser:
Try using a different web browser to see if the issue persists. Sometimes, browser-specific issues can affect uploads.

Verify File Size:
Confirm that the file size is indeed below 1 MB. Some image editing software may not accurately reflect the file size, so use GitHub's file size limit as a reference.

Check File Permissions:
Ensure that the file has the correct permissions for uploading. If the file is restricted, it might lead to upload failures.

Verify Internet Connection:
Ensure that your internet connection is stable. A weak or intermittent connection could lead to upload failures.

Update Browser:
Make sure your browser is up to date. Outdated browsers may have compatibility issues with certain features.

Try a Different Device:
If possible, try uploading the picture from a different device to rule out device-specific issues.

**Comment by arnab-bit** [comment, marked as answer]
Thanks! It has worked after using a different browser.
